src/lib/statistics/cardinality_estimator.cpp

```diff
@@ -748,7 +748,7 @@ std::shared_ptr<TableStatistics> CardinalityEstimator::estimate_operator_scan_pr
         if (input_table_statistics->row_count == 0 || sliced_histogram->total_count() == 0.0f) {
           selectivity = 0.0f;
         } else {
-          selectivity = sliced_histogram->total_count() / scan_statistics_object->total_count();
+          selectivity = sliced_histogram->total_count() / input_table_statistics->row_count;
```

Comment:
Absichtlich?

Reply:
This is introduced as an example change to see if old and new benchmark_all.sh detect changes equally well.

Reply:
This is the change from #2396.

Reply:
@Bensk1 remember to remove that.